### PR TITLE
Dice Roller feature

### PR DESCRIPTION
**Pull request by jjaw**

### CODE REVIEW

**Comment by jjaw** [comment]
trying to add a dice-roller feature, not sure if this npm package is still working

**Comment by dennischen0** [comment]
I get the following error:

```
dennischen@Denniss-MBP-10 ~/workspace/discord-hello-world-bot:(4h4m|git@51fef83!) ± npm start                                                                                                 ✹

> hello-world-bot@1.0.0 start /Users/dennischen/workspace/discord-hello-world-bot
> nodemon index.js

[nodemon] 2.0.4
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `node index.js`
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '../node_modules/rpg-dice-roller/dice-roller.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/dennischen/workspace/discord-hello-world-bot/commands/roll.js:1:20)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

**Comment by jjaw** [comment]
Can you try running npm install to update the package-lock.json ? I updated dependency in package.json

**Comment by dennischen0** [comment]
> Can you try running npm install to update the package-lock.json ? I updated dependency in package.json

you should commit the package-lock.json as well

**Comment by jjaw** [comment]
new PR should have everything commited

File: commands/roll.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+module.exports = {
+    name: 'roll',
+    alias: ['luck', 'r'],
+    description: 'Rolls a die',
+    execute(message, args) {
+        let dice = new DiceRoller.DiceRoller();
+        let input = args[0] ? args[0] : '1d7';
+        dice.roll(input);
+        let result = dice.log.shift();
+        let reply = `${message.author} rolled ${result.toString()}`;ls
```
Comment (dennischen0) [inline commands/roll.js:12]: lol

**Comment by dennischen0** [comment]
im going to add travis ci to catch these issues. once you get the ls out, should work and ill merge it in.

**Comment by dennischen0** [comment]
btw you should be able to run `npm test` to see if there are any issues

File: commands/roll.js (hunk 1 above)
Comment (dennischen0) [reply commands/roll.js:12]: I dont know how to fix it on your pr.

Comment (jjaw) [reply commands/roll.js]: fixed

**Pull request merged**